en_ewt-ud-test.conllu

```diff
@@ -31980,7 +31980,7 @@
 20	they	they	PRON	PRP	Case=Nom|Number=Plur|Person=3|PronType=Prs	21	nsubj	21:nsubj	_
 21	can	can	AUX	MD	VerbForm=Fin	18	advcl	18:advcl:as	_
 22	with	with	ADP	IN	_	24	case	24:case	_
-23	thier	they	PRON	PRP$	Number=Plur|Person=3|Poss=Yes|PronType=Prs|Typo=Yes	24	nmod:poss	24:nmod:poss	CorrectForm=they
+23	thier	they	PRON	PRP$	Number=Plur|Person=3|Poss=Yes|PronType=Prs|Typo=Yes	24	nmod:poss	24:nmod:poss	CorrectForm=their
```

Comment:
Lemma should be "their" (I presume you haven't added the check for this yet)

Reply:
That is indeed the case. This has now been fixed. Thanks.